tools/crucible_version

```diff
@@ -0,0 +1,5 @@
+# This should match the commit on package-manifest.toml
```

Comment:
Do we need to duplicate the version here?

Reply:
I went ahead and beefed up these scripts:

- They now parse all the files they modify, rather than having a duplicated version
- I moved a lot of this logic into a `./tools/update_helpers.sh`